### PR TITLE
Issue 997: rework notifications, refactor lists styles

### PR DESCRIPTION
**Pull request by MariiaNebesnova**

### CODE REVIEW

File: src/common/components/notification/useNotifications.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      </Button>
+    ) : null;
+
+  const cancelNotification = (action: Action) => () => {
+    if (Array.isArray(action)) {
+      action.forEach((_action) => {
+        dispatch(_action);
+      });
+    } else dispatch(action);
+    openNotification(messages().ACTION_CANCELLED, "info");
```
Comment (kostrubin) [inline src/common/components/notification/useNotifications.tsx:36]: Seems that this string needs a constant. Or will this also be done as part of a separate refactoring task?

File: src/common/styles/ant.less [hunk 2]
```diff
@@ -49,30 +49,7 @@
...
-      height: 100%;
-      display: flex;
-      align-items: center;
-    }
-
-    &-message {
-      margin-bottom: 0;
-    }
-  }
+  // position: absolute;
```
Comment (kostrubin) [inline src/common/styles/ant.less:52]: Forgotten comment?

File: src/common/components/notification/useNotifications.tsx (hunk 1 above)
Comment (MariiaNebesnova) [reply src/common/components/notification/useNotifications.tsx:36]: this string is checked by TS, it has a type `keyof NotificationInstance`, that comes from antd. There's no a constant for it in andt, but I'd like not to create my own constants, that will duplicate existing type.

File: src/common/styles/ant.less (hunk 2 above)
Comment (MariiaNebesnova) [reply src/common/styles/ant.less]: it is, thank you!

**Comment by MariiaNebesnova** [comment]
> I also noticed that we have some different ways to define colors (#fff, #ffffff, #ffffff50, rgb, rgba), so I’d suggest to unify this and use variables for colors and some other CSS properties.

agree with you, we definitely need them

**Review by kostrubin — APPROVED** [review]

**Pull request merged**